docs/REST API/rest-api.md

```diff
@@ -8805,6 +8806,42 @@ echo "hello"
 }
 ```
 
+### Modify script
+
+Modifies an existing script.
+
+`PATCH /api/v1/fleet/scripts/:id`
+
+##### Request body
+
+```json
+{
+  "script_content": "#!/bin/sh\\n\\n#!/usr/bin/env bash\\n\\nsudo systemsetup -settimezone Pacific/Ponape"
+}
+```
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                           |
+| ----            | ------- | ---- | --------------------------------------------          |
+| id              | integer | path | **Required**. The ID of the script to modify. |
+| script_contents | string  | body | The contents of the script to run. Only one of either `script_id` or `script_contents` can be included in the request; omit this parameter if using `script_id`. |
+
+#### Example
+
+`PATCH /api/v1/fleet/scripts/1`
+
+##### Default response
+
+`Status: 204`
+
+```json
+{
+  "script_content": "#!/bin/sh\\n\\n#!/usr/bin/env bash\\n\\nsudo systemsetup -settimezone Pacific/Ponape"
```

Comment:
I think this response should be the script data (in the "[Get script](https://fleetdm.com/docs/rest-api/rest-api#example-get-script2)" format versus the "Download script" format). 
```suggestion
{
  "id": 1,
  "team_id": null,
  "name": "script_1.sh",
  "created_at": "2023-07-30T13:41:07Z",
  "updated_at": "2023-07-30T13:41:07Z"
}
```

> Side note: I think it would actually be most consistent to nest the response data like this, but we're not doing that for other script endpoints so probably not worth it to change right now:
> ```json
> {
>   "script": {
>    "id": 1,
>    "team_id": null,
>    "name": "script_1.sh",
>    "created_at": "2023-07-30T13:41:07Z",
>    "updated_at": "2023-07-30T13:41:07Z"
>   }
> }
> ```